web/src/components/FeedbackToolbarItem.tsx

```diff
@@ -13,31 +13,47 @@ import ToolbarItem from './ToolbarItem'
 type FeedbackToolbarItemProps = {
   route: RouteType
   slug?: string
-  isInBottomActionSheet: boolean
 }
 
-const FeedbackToolbarItem = ({ route, slug, isInBottomActionSheet }: FeedbackToolbarItemProps): ReactElement => {
+const FeedbackToolbarItem = ({ route, slug }: FeedbackToolbarItemProps): ReactElement => {
   const { cityCode, languageCode } = useCityContentParams()
   const [isFeedbackOpen, setIsFeedbackOpen] = useState(false)
   const [isSubmitted, setIsSubmitted] = useState(false)
+  const [isPositive, setIsPositive] = useState<boolean | null>(null)
```

Comment:
It does not really make sense to have the same value in the parent and child as a state. I think it would make more sense to only have it in the parent and pass it (and the set function) down to the child.